idea/meta-storage-2/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": false
+  }
```

Comment:
What do you think about adding `tsconfig.build.json` file having `"include": ["src"]` prop for compiling? This way we won't have `tests` dir built during compilation

Reply:
In this case, I will exclude the test folder.